cypress/e2e/authentication/unlogged-user.cy.ts

```diff
@@ -156,7 +156,7 @@ describe('User Application', () => {
     //Just hit continue and check the route
     cy.contains('button', 'Continue').click();
     cy.get('#apply-job').should('exist');
-    cy.get('textarea[name=coverLetter]').eq(1).type('Example Cover Letter');
```

Comment:
@marjan-ara do we have more than 1 cover letter on this modal ?

Reply:
@jeyem No, before this PR we had (twice rendering), now we just have one modal, so this eq(1) is not needed anymore